src/views/Domains/FormFields/FormFieldsCreateDomains.vue

```diff
@@ -39,7 +38,7 @@
   const { value: edgeApplication, errorMessage: errorEdgeApplication } = useField('edgeApplication')
   const { setValue: setEdgeCertificate } = useField('edgeCertificate')
   const { value: mtlsIsEnabled } = useField('mtlsIsEnabled')
-  const { value: domainName } = useField('domainName')
+  useField('domainName')
```

Comment:
Why we are no longer assigning this to anything anymore?

Reply:
Porque não usaremos ele mais no FormFieldsCreateDomains eu só preciso que ela seja carregada para eu pegar o nome domain no handle.

Reply:
pode remover, pois o componente input field internamente usa esse hook para acessar o valor, não sendo mais necessário deixar ele aqui .